### ISSUE TITLE
State polls

### ISSUE THREAD

**Issue by tomkel** [McCartneyAC/average_of_polls#1]
Would it be worth time/effort to add tabs for state polls?

**Comment by McCartneyAC** [comment]
I'll look into it. If I can find a good data source that compiles all polls together by state, I'll make a module for it. Upkeep will be a pain though. I think you're right that it could be essential, especially if we wanted to do a 'polls-only' model of who is going to win the nomination.

**Comment by McCartneyAC** [comment]
Okay, added the first four primaries--doesn't seem like there's a big database for the remaining 53 states/territories. I'll update the site when I get to my desktop tonight.

**Comment by tomkel** [comment]
Yeah the RCP early primary polls are pretty much useless with how sensitive the model is

**Comment by McCartneyAC** [comment]
Nate Silver's model (yet unpublished for this season) is approx 9,000 lines of stata code, so that should give some indication of what exactly we're working with in terms of complexity.

I can't even get the damn thing to control for house effects. Their primary polling model is coming soon and will do this at which point this repo will be a bit redundant/obsolete.

**Comment by McCartneyAC** [comment]
So I built a module for these state primaries and loaded it locally last night. It'll go live probably by this weekend. Anyway, the ratio of noise to signal in the state primary/caucuses is really high and the smoothing algorithm doesn't really know how to handle it. It's a cool idea but I don't think it'll be workable until right before each state's primary, if ever. 

Working on a few other neat ideas though. Stay tuned.

**Comment by tomkel** [comment]
Does RCP use likely voters / all voters / registered voters? I've noticed different numbers than 538 sometimes.

Also what do you think about RCP excluding some pollsters? Ipsos for example

**Comment by tomkel** [comment]
Also I think 538 just updated their pollster ratings, so the house effects probably are different now too.

Yeah, just checked, they are all different.

**Comment by McCartneyAC** [comment]
(1) RCP uses different things because polls report different things... this is a source of measurement error that is systematic and one I would like to eliminate with mixed effects if / when I have time to sit down and figure it out. 
(2) in general, RCP is reporting democratic primary voters.

**Comment by McCartneyAC** [comment]
(3) will get into house effects the next time I sit down for sustained work on this project. Thanks!
